event-svc/src/event/service.rs

```diff
@@ -141,44 +193,15 @@ impl EventService {
         Ok(())
     }
 
-    /// Currently only verifies that the event parses into a valid ceramic event.
-    /// In the future, we will need to do more event validation (verify all EventID pieces, hashes, signatures, etc).
-    pub(crate) async fn parse_discovered_event(
-        item: &ReconItem<EventId>,
-    ) -> Result<EventInsertable> {
-        let (cid, parsed_event) =
-            unvalidated::Event::<Ipld>::decode_car(item.value.as_slice(), false)
-                .map_err(Error::new_app)?;
-
-        Ok(EventInsertable::new(
-            item.key.to_owned(),
-            cid,
-            parsed_event,
-            false,
-        )?)
-    }
-
-    async fn validate_signed_events(
-        events: Vec<EventInsertable>,
-    ) -> Result<(Vec<EventInsertable>, Vec<InvalidItem>)> {
-        // TODO: IMPLEMENT THIS
-        Ok((events, Vec::new()))
-    }
-
-    async fn validate_time_events(
-        events: Vec<EventInsertable>,
-    ) -> Result<(Vec<EventInsertable>, Vec<InvalidItem>)> {
-        // TODO: IMPLEMENT THIS
-        Ok((events, Vec::new()))
-    }
-
-    pub(crate) async fn validate_events(
+    pub(crate) async fn insert_events(
+        &self,
         items: &[ReconItem<EventId>],
-    ) -> Result<(Vec<EventInsertable>, Vec<InvalidItem>)> {
+        source: DeliverableRequirement,
```

Comment:
yeah, it's not right.. iI called it source originally because it was like API/recon and then we made the deliverable enum instead and I didn't rename the variable. but it's simple to change as it's just internal to the function.